d2target/sqltable.go

```diff
@@ -41,14 +43,22 @@ func (c SQLColumn) Texts(fontSize int) []*MText {
 }
 
 func (c SQLColumn) ConstraintAbbr() string {
-	switch c.Constraint {
-	case "primary_key":
-		return "PK"
-	case "foreign_key":
-		return "FK"
-	case "unique":
-		return "UNQ"
-	default:
-		return ""
+	var abbrs []string
+
+	for _, constraint := range c.Constraint {
+		var abbr string
+
+		switch constraint {
+		case "primary_key":
+			abbr = "PK"
+		case "foreign_key":
+			abbr = "FK"
+		case "unique":
+			abbr = "UNQ"
+		}
+
+		abbrs = append(abbrs, abbr)
 	}
+
+	return strings.Join(abbrs, " ")
```

Comment:
what if we join with comma? `, `. i think that could help distinguish multiple better